locales/fluent/base/en.ftl

```diff
@@ -1,8 +1,44 @@
 ### Base localization / generic strings
+### Messages should only go into this file if they are widely used,
+### or are particularly important to localize.
 
-# Special message that shows whenever a message is still being loaded
+## Terms
+
+-service-name = Wikijump
```

Comment:
What do leading dashes mean?

Reply:
These are "terms", which I would think of as being like environment variables. They have a bunch of uses, especially because they support stuff like attributes and placeholders that can be given by other messages.

Reply:
Makes sense!